WashingBot.js

```diff
@@ -46,7 +46,7 @@ function setupUI(){
 
     txtConnection = app.CreateText("", 0.8, 0.2, "Multiline");
     lay.AddChild(txtConnection);
-    txtConnection.SetText("Waiting...");
+    txtConnection.SetText("Slack notifications disabled");
```

Comment:
It's there because if integration is enabled, it will instantly change to "Waiting", the user will never see it.
But if it's disabled, it will remain as it was defined, for ever. It was a simple way of solving it. Of course it could be inside setupSlackBot function.

Do you think it's important to move it?